downstream/modules/platform/con-known-proxies.adoc

```diff
@@ -1,17 +1,20 @@
 
 [id="con-known-proxies_{context}"]
 
-= About known proxies
+= Known proxies
 
 
 [role="_abstract"]
 
-When Tower is configured with `REMOTE_HOST_HEADERS = ['HTTP_X_FORWARDED_FOR', 'REMOTE_ADDR', 'REMOTE_HOST']`, it assumes that the value of `X-Forwarded-For` has originated from the proxy/load balancer sitting in front of Tower. In a scenario where Tower is still reachable without use of the proxy/load balancer or when the proxy does not validate the header, `X-Forwarded-For` can be spoofed fairly easily to fake the originating IP addresses. Using `HTTP_X_FORWARDED_FOR` in the `REMOTE_HOST_HEADERS` setting poses a vulnerability that essentially gives users access to certain resources that they should not have.
+When {ControllerName} is configured with `REMOTE_HOST_HEADERS = ['HTTP_X_FORWARDED_FOR', 'REMOTE_ADDR', 'REMOTE_HOST']`, it assumes that the value of `X-Forwarded-For` has originated from the proxy/load balancer sitting in front of {ControllerName}.
+In a scenario where {ControllerName} is still reachable without use of the proxy/load balancer or when the proxy does not validate the header, `X-Forwarded-For` can be spoofed fairly easily to fake the originating IP addresses.
```

Comment:
I was never sure about the word 'spoofed' it doesn't sound particularly technical!

Reply:
You're right: that whole sentence could do with a re-write.

Reply:
@ianf77 
Could you please review the new change I made?